### ISSUE TITLE
Compilation error in Xcode 13.3 RC

### ISSUE THREAD

**Issue by klundberg** [pointfreeco/swift-parsing#180]
I'm playing with using the latest release of swift-parsing, but for some reason I'm getting this compiler error on Xcode 13.3 RC:

```
<derived-data-folder>/SourcePackages/checkouts/swift-parsing/Sources/Parsing/ParsingError.swift:316:11: error: 'subscript(_:)' is unavailable
        : originalInput.base[originalInput.startIndex..<remainingInput.startIndex]
          ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
Swift.MutableCollection:6:23: note: 'subscript(_:)' has been explicitly marked unavailable here
    @inlinable public subscript(bounds: Range<Self.Index>) -> Slice<Self> { get set }
                      ^
```

I'm integrating it as a swift package directly through xcode. I'm not sure if there's differences in how I could integrate it that would make a difference.

I haven't tried with Xcode 13.2.1 yet, but I imagine the results must be different.

**Comment by klundberg** [comment]
Ah, I found this commit shortly after filing this: https://github.com/pointfreeco/swift-parsing/commit/2c1b4784d63b8a490d7b0ef6e4ab836dbab4c845. I'll base my work off of main for now until the next release.

**Comment by tgrapperon** [comment]
Hello, this is fixed by #146. There is no release including these changes yet, but you can maybe update your dependencies and point on `main` to make it build in the meantime.
